src/utils/hooks.ts

```diff
@@ -63,3 +64,25 @@ export const useMount = (fn: () => void) => {
     //eslint-disable-next-line
   }, []);
 };
+
+export const useHentMalInformasjon = (path: string) => {
+  const [filstorrelse, settFilstorrelse] = useState(0)
+  const [filtype, settFiltype] = useState('')
+
+  useEffect(() => {
+    const hentFilInformasjon = (url: string) => {
+      let filBlob;
+      fetch(url).then((res) => {
```

Comment:
Usikker på om det har så mye å si, men vi bruker nok mest `axios` istedenfor fetch for nettverkskall 🤓

Reply:
Har byttet til å bruke Axios nå.